src/store/modules/Control/PoweCapStore.js

```diff
@@ -0,0 +1,43 @@
+import api from '../../api';
+
+const PowerCapStore = {
+  namespaced: true,
+  state: {
+    powerCapData: null,
+    powerCapValue: 'Not enabled'
+  },
+  getters: {
+    powerCapData(state) {
+      return state.powerCapData;
+    },
+    powerCapValue(state) {
+      return state.powerCapValue;
+    }
+  },
+  mutations: {
+    setPowerCapData(state, powerCapData) {
+      state.powerCapData = powerCapData;
+    },
+    setPowerCapValue(state, powerCapValue) {
+      state.powerCapValue = powerCapValue;
+    }
+  },
+  actions: {
+    getPowerCapData({ commit }) {
+      api
+        .get('/xyz/openbmc_project/control/host0/power_cap')
+        .then(({ data }) => {
+          const powerCapData = data.data;
```

Comment:
Are we going to use object destructuring or maintain consistency in showing the response's data structure? I feel like its more readable when not using object destructuring.